modules/virt-configuring-storage-class-bootsource-update.adoc

```diff
@@ -11,7 +11,11 @@ You can override the default storage class by editing the `HyperConverged` custo
 
 [IMPORTANT]
 ====
-Boot sources are created from storage using the default storage class. If your cluster does not have a default storage class, you must define one before configuring automatic updates for custom boot sources.
+Boot sources are created using the default storage class. When the default storage class changes, old boot sources are automatically updated using the new default storage class.
+
+If boot source images were stored as VolumeSnapshots and both the cluster default and `virt-default` storage class have been unset, the VolumeSnapshots will be cleaned up and new DataVolumes will be created. However they will not start importing until a default storage class is set.
+
```

Comment:
What do you think about adding this new module right here?
`include::modules/virt-configuring-default-and-virt-default-storage-class.adoc[leveloffset=+2]`

So the story flow is:
- Title: Configuring a storage class for the boot source images 
- Boot sources need a default sc, here's how to set it
- You can set the sc in HCO, here's how